packages/legacy-client/src/ontology-runtime/ontologyProvider/calls/getAttachment.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2023 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { createOpenApiRequest, type ThinClient } from "@osdk/api";
+import { getAttachmentContent } from "@osdk/gateway/requests";
+import {
+  AttachmentsErrorHandler,
+  handleAttachmentsError,
+} from "../ErrorHandlers";
+import type { AttachmentsError } from "../Errors";
+import type { Result } from "../Result";
+import { wrapResult } from "./util/wrapResult";
+
+export async function getAttachment(
+  client: ThinClient<any>,
```

Comment:
I need to make a pass but this actually should be `ThinClient<OntologyDefinition<any>>`